ext/ffi/lib.rs

```diff
@@ -791,6 +798,9 @@ fn make_sync_fn<'s>(
       .collect::<Vec<_>>();
     // recv
     args.insert(0, fast_api::Type::V8Value);
+    if needs_unwrap {
+      args.push(fast_api::Type::TypedArray(fast_api::CType::Int32));
```

Comment:
From V8 sources:
```cpp
 #define TYPED_ARRAY_C_TYPES(V)
   V(int32_t, kInt32)
   V(uint32_t, kUint32)
   V(int64_t, kInt64)
   V(uint64_t, kUint64)
   V(float, kFloat32)
   V(double, kFloat64)
```
ie. It looks like Uint64 would be an okay type a well, or Int64. Would those be an option instead?

Reply:
not sure why it's defined that way but BigUint64Array deopt fast calls.

Reply:
Huh. Well that's dumb.

Reply:
I tried setting the array item type to `Int64` and passing in the `BigInt64Array` created in `00_ffi.js` and it seemed to work as expected. Is it possible you might've set the array item type to `Uint64` but passed in a `BigInt64Array`?